src/crimson/net/ProtocolV2.cc

```diff
@@ -1118,11 +1118,6 @@ ProtocolV2::handle_existing_connection(SocketConnectionRef existing_conn)
                  existing_proto->client_cookie,
                  existing_proto->server_cookie);
 
-  if (existing_proto->state == state_t::CLOSING) {
```

Comment:
see https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst#3-describe-your-changes . i'd recommend use the imperative mood in the commit title.

Reply:
reworded, thanks.